The test suite seems to not handle our userland hg repository in some cases.

Should be reported upstream.

--- setuptools_scm-8.3.1/testing/test_basic_api.py
+++ setuptools_scm-8.3.1/testing/test_basic_api.py
@@ -137,6 +137,7 @@ setup(name="myscm", use_scm_version={"fa
     assert res.stdout == "12.34"
 
 
+@pytest.mark.skip("doesn't work with our component archives")
 def test_get_version_blank_tag_regex() -> None:
     with pytest.warns(
         DeprecationWarning, match="empty regex for tag regex is invalid, using default"
--- setuptools_scm-8.3.1/testing/test_mercurial.py
+++ setuptools_scm-8.3.1/testing/test_mercurial.py
@@ -147,6 +147,7 @@ def test_version_in_merge(wd: WorkDir) -
 
 
 @pytest.mark.issue(128)
+@pytest.mark.skip("doesn't work in our infrastructure")
 def test_parse_no_worktree(tmp_path: Path) -> None:
     config = Configuration()
     ret = parse(os.fspath(tmp_path), config)
